### DISCUSSION TITLE
render_template_as_native_obj=True convert string into tuple

### DISCUSSION THREAD

**Discussion by yilinlesley** [apache/airflow#31770] [Q&A]
I was trying to test one of the customized SSH operator with render_template_as_native_obj=True and noticed that with that set to true, it automatically converts a string into a tuple:
Originally: '"%let ' +'SomeFolder:path1/path2/path3;" + ','+'"%let '+'SomeFolder:path1/path2/path3;"'
Now: ('%let SomeFolder:path1/path2/path3;', '%let SomeFolder:path1/path2/path3;')
I have checked the type of the 'Now' version and it returns 'tuple'.

Does anyone know why does this kind of conversion happen?

**Comment by potiuk** [comment]
This is used: https://jinja.palletsprojects.com/en/3.0.x/nativetypes/#jinja2.nativetypes.NativeEnvironment you might ask details in Jinja dsicussion lists
